Userland/Applications/PixelPaint/IconBag.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022, Dylan Katz <dykatz@uw.edu>
```

Comment:
I'm flattered, but you don't need to give me the copyright for this. Ditto for the other file.

Reply:
No worries, wasn't sure how you'd feel!